client/sniffer/src/test/resources/create_test_nodes_info.bash

```diff
@@ -60,7 +60,7 @@ node${attr}.dummy:  everyone_has_me
 node${attr}.number: ${node:1}
 node${attr}.array:  [${node:0:1}, ${node:1}]
 http.port:          ${http_port}
-transport.tcp.port: ${transport_port}
+transport.port: ${transport_port}
```

Comment:
this  script runs old versions of elasticsearch (2.0.0, 5.0.0, 6.0.0) so should not be updated